db/schema.rb

```diff
@@ -246,6 +247,7 @@
     t.text "role", default: "publisher"
     t.datetime "javascript_last_detected_at"
     t.datetime "default_currency_confirmed_at"
+    t.boolean "uphold_disabled", default: false, null: false
```

Comment:
Would `exclude_from_payout` or something similar be better here?  We want to prevent payouts regardless of the wallet provider